EasyPostTest/ShipmentTest.cs

```diff
@@ -310,5 +310,39 @@ public class ShipmentTest {
             ShipmentList nextShipmentList = shipmentList.Next();
             Assert.AreNotEqual(shipmentList.shipments[0].id, nextShipmentList.shipments[0].id);
         }
+
+        //Smart rate
+        [TestMethod]
+        public void ApiCallIsNotNull()
+        {
+            Shipment shipment = Shipment.Create(parameters);
+            List<Smartrate> smartRateResult = shipment.GetSmartrates();
+            Assert.IsNotNull(smartRateResult);
+        }
+
+        [TestMethod]
+        public void TestGetSmartrates()
+        {
+            Shipment shipment = Shipment.Create(parameters);
+            List<Smartrate> smartRateResult = shipment.GetSmartrates();
+            //Make sure shipment id from smartrate is the same as the created one
+            Assert.AreEqual(shipment.rates[0].id, smartRateResult[0].id);
+            Assert.IsNotNull(shipment.rates);
+        }
+
+        //This API call changes each time. Replicated from the python version. Will fail since percentiles are changing each time. Will check for type instead.
```

Comment:
Let's change this to a `TODO` comment so we can come back to it later. Something along the lines like we used in the PHP library:

`//TODO: Once we've added `vcr`, assert that the following values actually match an integer`